src/components/Formik/FormikCheckboxInput/FormikCheckboxInput.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {
+  FC,
+  ChangeEvent,
+  FocusEvent,
+} from 'react';
+import CheckboxInput from '../../CheckboxInput/CheckboxInput';
+
+interface FormikCheckboxInputProps {
+  field: {
+    id: string;
+    label: string;
+    name: string;
+    onBlur: (event: FocusEvent<HTMLElement>) => void;
+    onChange: (event: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
should these be optional?

Reply:
@juanfabrega

Reply:
@ericyliu the prop structure here is based on what the formik `<Field>` component actually passes down, we are only adhering to that structure and then spreading any of our custom props.

@JCarmine I noticed we added some props to the `field` object from what we had before, did you look at any docs that perhaps said these were missing? Formik uses the `name` prop to id a field, but I don't believe it utilizes the `id` or `label` those actually get passed down to our component as part of the prop spread. If you want to outline them individually we can do that outside of the `field` object.